src/main/java/org/sagebionetworks/web/client/view/ProfileViewImpl.java

```diff
@@ -267,9 +273,15 @@ public ProfileViewImpl(
     teamSearchButton.addClickHandler(event ->
       presenter.goTo(new TeamSearch(teamSearchTextBox.getValue()))
     );
+    teamSearchButtonSmall.addClickHandler(event ->
+      presenter.goTo(new TeamSearch(teamSearchTextBox.getValue()))
+    );
```

Comment:
Can you re-use the handlers? e.g. 

For all of the handler changes in this PR.

```suggestion
	ClickHandler teamSearchHandler = event -> presenter.goTo(new TeamSearch(teamSearchTextBox.getValue()));
    teamSearchButton.addClickHandler(teamSearchHandler);
    teamSearchButtonSmall.addClickHandler(teamSearchHandler);
```